acrobat/blocks/dc-converter-widget/dc-converter-widget.js

```diff
@@ -113,7 +113,16 @@ const verbRedirMap = {
   'number-pages': 'number',
 };
 
+const exhLimitCookieMap = {
+  'to-pdf': 'p_ac_cr_p_c',
+  'pdf-to': 'p_ac_ex_p_c',
+  'compress-pdf': 'p_ac_cm_p_ops',
+  'rotate-pages': 'p_ac_or_p_c',
+  createpdf: 'p_ac_cr_p_c',
```

Comment:
hate to add more logic but we should use p_ s_ and d_ for prod, stage and dev, respectively.  Perhaps remove the p_ from this map and prepend the right prefix to `limitCookie` at L220